packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion-group/outlined.ts

```diff
@@ -0,0 +1,23 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `outlined` variant for component `suggestion group`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function suggestionGroupOutlined(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    outlined: {
+      backgroundColor: theme('colors.neutral.0'),
+      borderWidth: theme('borderWidth.1'),
+      borderColor: 'var(--suggestion-group-color-50)',
+      '&:hover': {
+        backgroundColor: theme('colors.neutral.10')
+      },
+      gap: theme('spacing.4')
+    }
```

Comment:
issue: arren't outlined and ghost variants missing the padding left/right styles depending on their size? 
![image](https://user-images.githubusercontent.com/68222542/209783329-52d6f45c-7878-45aa-b644-06e359379ead.png)

Reply:
you are right again, good catch!